src/FilterButtons/DeleteDoneBtn.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import UseTodos from '../hooks/useTodos';
+import { useSnackbar } from 'notistack';
+import UseFilterTodos from '../hooks/useFilterTodos';
+
+const DeleteDoneBtn = () => {
+
+    const { tasks, setTasks } = UseTodos();
+    const { setFilterTasks } = UseFilterTodos();
+    const { enqueueSnackbar } = useSnackbar();
+
+    const handleDeleteDone = () => {
+        try {
+            const updatedTasks = tasks.filter(task => task.isChecked === false);
+            setTasks(updatedTasks);
+            setFilterTasks(updatedTasks);
```

Comment:
the filter tasks need to be depandance on the original tasks. so you dont need to setFilterTasks every time you set setTasks